packaging/build-release-zip.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
```

Comment:
I think we should add the 'set -eu' command, so that when there is a failure, errors are not ignored, and we bundle the right code

Reply:
done in c8ff75b